packages/graphback-core/src/runtime/CRUDService.ts

```diff
@@ -105,7 +104,7 @@ export class CRUDService<T = any> implements GraphbackCRUDService<T>  {
     }
   }
 
-  public subscribeToCreate(filter: any, context: GraphbackContext): AsyncIterator<T> | undefined {
+  public subscribeToCreate(filter: any, context: GraphbackContext): AsyncIterator<Type> | undefined {
```

Comment:
```suggestion
  public subscribeToCreate(filter: QueryFilter<Type>, context: GraphbackContext): AsyncIterator<Type> | undefined {
```

filter missing type?

Reply:
I would not put this here as we do not support any filtering. So if I put anything it will be instantly breaking change where we do not know what to filter and how

Reply:
Okay, very good point.